src/context.ts

```diff
@@ -107,12 +97,12 @@ export class Context {
 
   get message() {
     if (!('message' in this.update)) return undefined
-    return deunionize(this.update.message)
+    return this.update.message
```

Comment:
why not use optional chaining.

```ts
return this.update?.message
```

see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
We have to narrow down `Update` before we can access message, as it is not always present on an update.

Otherwise, we get
```
Property 'message' does not exist on type 'Update'.
  Property 'message' does not exist on type 'CallbackQueryUpdate'.ts(2339)
```